src/main/resources/application.yml

```diff
@@ -9,3 +9,61 @@ file-upload:
   sourceBasePath: src/test/resources
   targetBasePath: ready_to_agent
   filePrefixForLocalProcessing: local_file_prefix
+  globus:
```

Comment:
Do we need these lines in the main `application.yml` file

Reply:
i added them so i can run this service locally.

Reply:
I understand that, but do we need it in the repository?